src/Compilers/Test/Core/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -1102,6 +1160,22 @@ bool isLongLivedCaptureReferenceSyntax(SyntaxNode captureReferenceSyntax)
                                         return true;
                                     }
                                     break;
+
+                                case CSharp.SyntaxKind.ConditionalAccessExpression:
+                                    if (((CSharp.Syntax.ConditionalAccessExpressionSyntax)syntax.Parent).Expression == syntax)
+                                    {
+                                        return true;
+                                    }
+                                    break;
+                                case CSharp.SyntaxKind.SimpleAssignmentExpression:
+                                    if (((CSharp.Syntax.AssignmentExpressionSyntax)syntax.Parent).Left == syntax
+                                        && syntax.Parent.Parent is InitializerExpressionSyntax { Parent: CSharp.Syntax.ObjectCreationExpressionSyntax })
```

Comment:
>&& syntax.Parent.Parent is InitializerExpressionSyntax { Parent: CSharp.Syntax.ObjectCreationExpressionSyntax }) [](start = 40, length = 112)

Would it be possible to tighten the condition by also checking something other than syntax? Perhaps check symbols involved?

Reply:
I do not think we should recognize this as long lived, it doesn't escape the statement.

---
In reply to: [530036537](https://github.com/dotnet/roslyn/pull/49534#discussion_r530036537) [](ancestors = 530036537)

Reply:
Could isObjectOrArrayInitializerInitializedObjectTarget help for this scenario?

---
In reply to: [530040111](https://github.com/dotnet/roslyn/pull/49534#discussion_r530040111) [](ancestors = 530040111,530036537)